tests/acceptance/30_generic_methods/group_absent.cf

```diff
@@ -0,0 +1,76 @@
+#######################################################
+#
+# Test checking if a group exist
+#
+#######################################################
+
+bundle common acc_path
+{
+  vars:
+    "root" string => getenv("NCF_TESTS_ACCEPTANCE", 1024);
+}
+
+body common control
+{
+      inputs => { "${acc_path.root}/default.cf.sub", "${acc_path.root}/default_ncf.cf.sub", "@{ncf_inputs.default_files}" };
+      bundlesequence  => { configuration, default("$#this.promise_filename}") };
+      version => "1.0";
+}
+
+#######################################################
+
+bundle agent init
+{
+  vars:
+    "group1"      string => "group1";
+    "group2"      string => "group2";
+
+  commands:
+    "/usr/sbin/groupadd ${init.group2}" handle => "a2";
+}
+
+#######################################################
+
+bundle agent test
+{
+  methods:
+    "ph1" usebundle => group_absent("${init.group1}");
+    "ph2" usebundle => group_absent("${init.group2}");
+}
+
+#######################################################
+
+bundle agent check
+{
+  vars:
+    "group1_id" int => getgid("${init.group1}");
+    "group2_id" int => getgid("${init.group2}");
+
+  classes:
+    "group1_ok" not => isvariable("group1_id");
+    "group2_ok" not => isvariable("group2_id");
```

Comment:
why not groupexists ?
https://docs.cfengine.com/docs/3.5/reference-functions-groupexists.html